nym-vpn-apple/Services/Sources/Services/NymLogger/LogFileManager.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+import Constants
+
+public final class LogFileManager {
+    public static let shared = LogFileManager()
+
+    init() {
+        configure()
+    }
+
+    public var fileHandle: FileHandle?
```

Comment:
`fleHandle` leaks outside of `LogFileManager`

If we are going to share it with other `FileLogHandler` types then I think it would be sensible to handle all io in here and make sure we use a dispatch queue to serialize calls.

As a template what comes to my mind:

```swift
private let queue = DispatchQueue(label: "..")

public func write(str: String) {
  queue.async { 
     // open or create file if file handle is nil
     self.fileHandle = ...
  }
}

public func deleteFile() {
  queue.async {
    // delete file
    // release file handle
    self.fileHandle = nil
  }
}
```